web-console/src/views/query-view/query-view.tsx

```diff
@@ -136,6 +137,7 @@ export class QueryView extends React.PureComponent<QueryViewProps, QueryViewStat
   private readonly queryManager: QueryManager<QueryWithContext, QueryResult>;
 
   private readonly queryInputRef: RefObject<QueryInput>;
+  // private id = 0;
```

Comment:
Can this be deleted?

Reply:
yes i will delete that, thank you for spotting it !